toolchain/check/impl_lookup.cpp

```diff
@@ -628,16 +680,16 @@ auto EvalLookupSingleImplWitness(Context& context, SemIR::LocId loc_id,
   bool query_is_concrete =
       QueryIsConcrete(context, query_self_const_id, query_specific_interface);
 
-  // If we have a symbolic witness in the self query, then the query can not be
-  // concrete: the query includes a symbolic self value.
+  // If we have a symbolic witness in the self query, then the query can not
+  // be concrete: the query includes a symbolic self value.
   CARBON_CHECK(!self_facet_provides_witness || !query_is_concrete);
 
-  // If the self value is a (symbolic) facet value that has a symbolic witness,
-  // then we don't need to do impl lookup, except that we want to find any final
-  // impls to return a concrete witness if possible. So we limit the query to
-  // final impls only in that case. Note as in the CHECK above, the query can
-  // not be concrete in this case, so only final impls can produce a concrete
-  // witness for this query.
+  // If the self value is a (symbolic) facet value that has a symbolic
+  // witness, then we don't need to do impl lookup, except that we want to
+  // find any final impls to return a concrete witness if possible. So we
+  // limit the query to final impls only in that case. Note as in the CHECK
+  // above, the query can not be concrete in this case, so only final impls
+  // can produce a concrete witness for this query.
```

Comment:
Is it worth avoiding rewrites of comments like this, to preserve git history? (this whole file has changes)

Reply:
This happens to me too much :/ I will fix it, it's not intended.